manifests/install.pp

```diff
@@ -10,6 +10,13 @@
     before => [Service['qpidd'],Package['qpid-tools']],
   }
 
+  if $qpid::auth {
+    package { 'cyrus-sasl-plain':
+      ensure => present,
+      before => [Service['qpidd'],Package['qpid-tools']],
```

Comment:
Why is this needed? I see it it was already there but I really wonder why it needs to be. `Service['qpidd']` should be included in `init.pp` already. Also I don't see why this package should be installed before `qpid-tools`.

Reply:
https://github.com/theforeman/puppet-qpid/pull/150 should clean that up.